.github/workflows/publish-commit-bottles.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     container:
       image: ghcr.io/homebrew/ubuntu22.04:master
     permissions:
-      pull-requests: write
+      pull-requests: ${{inputs.commit_bottles_to_pr_branch && 'write' || 'read'}}
```

Comment:
🤷🏻 IMO it's fine to just leave as `write` unconditionally.

Reply:
Yea, I guess it doesn't hurt. I'll move it up along with the `permissions` settings for `contents` in a follow-up.